en/deploy-br-federation-manager.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Deploy BR Federation Manager on Kubernetes
+summary: Learn how to deploy BR Federation Manager on Kubernetes.
+aliases: ['/docs/tidb-in-kubernetes/dev/deploy-br-federation-manager/']
+---
+
+# Deploy BR Federation Manager on Kubernetes
+
```

Comment:
Please explain what is BR Federation Manager. As this is a brand new term for our user documentation, many might wonder what it is and how it is different from BR.